Code/Engine/Foundation/Application/Implementation/MainLoop.cpp

```diff
@@ -26,6 +26,7 @@ void ezRun_MainLoop(ezApplication* pApplicationInstance)
 {
   while (pApplicationInstance->Run() == ezApplication::Execution::Continue)
   {
+    EZ_PROFILER_END_FRAME;
```

Comment:
@C-Core don't we already have some end frame marker? I thought we need this for the GPU stuff.

Reply:
We have end frame in GALDevice that is used for the GPU profiling scopes, but it wouldn't make sense to add the tracy end frame marker there. We could use the ezProfilingSystem::StartNewFrame function though and add the tracy macro in there. Then all the changes would be lokal to the profiling system.